### DISCUSSION TITLE
Lacking feature

### DISCUSSION THREAD

**Discussion by ZagkasD** [Rem0o/FanControl.Releases#1368] [Ideas]
The software is lacking a very important feature, I believe.

With the mix graph, you don't actually use the temperatures of the components as inputs, but rather the values of the other choosen graphs. Which means, you can't create a separate graph for your case fans, which will run independently from the CPU/GPU fans and take the max TEMP of those two as input.

The case fans will have to spin at the speed of either the cpu or gpu, which is a big problem. Consider the stock cpu coolers which most people use. Those are smaller than the standard case fans and they spin faster. If you match your case fans to the cpu fan, you'll end up with a noisy computer.

The solution would be to choose temperatures as inputs in the graphs. Tha way, you can choose the max temp of either gpu or cpu and with that input, choose the value from the case fans graph.

**Comment by Rem0o** [comment]
There are two ways to do what you want.

A: Create a mix sensor, put CPU and GPU in there, use that as your case fan graph input.
B: Create a total of 3 fan curves for your case fans, seperate from your CPU and GPU fans : 1 cpu temp based, 1 gpu temp based, then a mix of those 2.

Personally, I use option B. I use 2 other fan curves (not shown) to control my GPU fan and CPU fan. Those are just for my case fans.  Only the mix fan curve is assigned, the other 2 are just used for the mix.
![image](https://user-images.githubusercontent.com/14118956/207445424-80f524b1-9add-416b-9f87-93f0b10d547a.png)

**Reply by ZagkasD** [reply]
That is actually a very simple solution that I didn't spend the time to think.

Thanks, it works.
